VS/VehicleSignals.rdf

```diff
@@ -4024,6 +4030,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;RightIndicatorLight">
 		<rdfs:subClassOf rdf:resource="&auto-vs;ActuableSignal"/>
+		<rdfs:label xml:lang="en">Right indicator light</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.